src/stk500v2.c

```diff
@@ -294,7 +301,10 @@ void stk500v2_setup(PROGRAMMER * pgm)
   memset(pgm->cookie, 0, sizeof(struct pdata));
   PDATA(pgm)->command_sequence = 1;
   PDATA(pgm)->boot_start = ULONG_MAX;
-  PDATA(pgm)->xtal = STK500V2_XTAL;
+  if (str_starts(pgmid, "scratchmonkey"))
```

Comment:
Crisper and clearer in one line, but that's a matter of taste (and it's your decision).

Reply:
ok, looks good 
```c
PDATA(pgm)->xtal = (str_starts(pgmid, "scratchmonkey")) ? SCRATCHMONKEY_XTAL : STK500V2_XTAL;
```

Reply:
Looks so much neater, don't you agree? You can even remove the parentheses for the test and the spaces before `?` and `:` to crisp it up:
```
  PDATA(pgm)->xtal = str_starts(pgmid, "scratchmonkey")? SCRATCHMONKEY_XTAL: STK500V2_XTAL;
```